azure-pipelines.yml

```diff
@@ -14,6 +14,7 @@ jobs:
     os: windows
     architecture: x64
     frameworks: [net35, net452, net472, net48, netcoreapp3.0, netcoreapp3.1, net5.0, net6.0, net7.0, net8.0]
+    buildConfigurations: [ReleaseFat, ReleaseLite]
```

Comment:
Seems this line breaks the Azure tests with:
/azure-pipelines.yml: (Line: 17, Col: 15, Idx: 394) - (Line: 17, Col: 15, Idx: 394): While scanning a simple key, could not find expected ':'.